include/motor-control/core/brushed_motor/brushed_motor_interrupt_handler.hpp

```diff
@@ -55,28 +57,30 @@ class BrushedMotorInterruptHandler {
                 limit_switch_triggered()) {
                 homing_stopped();
             } else {
-                tick_count++;
-                if (!should_continue()) {
+                if (is_idle) {
```

Comment:
We may need a holdoff here, a minimum time before we start checking `is_idle` so we give the motor time to accelerate and actually start moving.